src/wallet/rpcwallet.cpp

```diff
@@ -2716,7 +2716,7 @@ UniValue mint(const UniValue& params, bool fHelp)
 
     if (fHelp || params.size() != 1)
         throw std::runtime_error(
-            "mint amount\n"
+            "mint <amount>(could not smaller than 0.1)\n"
             "\nAutomatically choose denominations to mint by amount\n" +
             HelpRequiringPassphrase());
```

Comment:
We need to add more description too. Take a look at `sendmany` RPC for some example.